src/main/java/com/checkmarx/flow/custom/ServiceNowTracker.java

```diff
@@ -384,13 +391,12 @@ private Incident getCreateIncident(ScanResults.XIssue resultIssue, ScanRequest r
      * @param request ScanRequest object
      * @return Incident object
      */
-    private Incident getCloseIncident(ScanRequest request) {
+    private Incident getCloseIncident() {
         Incident incident = new Incident();
-        incident.setSysId(request.getId());
```

Comment:
If you do not set the incident ID, how will ServiceNow know what is the ticket to close? You left it on your change of the `updateIncidentFromIssue()`, it shouldn't be removed from this one, maybe it should have been the `issue.getId()` and not the `request.getId()` as it's done in the `updateIncidentFromIssue()` (line 323) method?

Reply:
@cx-scord the ticket ID is provided in the URL Path, not in the body